_includes/assets/css/styles.css

```diff
@@ -401,7 +403,7 @@ main ul, main ol {
 COMPONENTS
 ========================================================================== */
 .title-bar {
-  background-color: #CE132D;
+  background-color: green;
```

Comment:
Should this be a HEX code or a variable? Green will be different in every browser.

Reply:
Argh, this was test css that I hadn't removed. Cheers!